learn/configuration/configuring_index_settings.mdx

```diff
@@ -28,37 +28,39 @@ Using the menu on the left-hand side, click on "Attributes":
 
-Click on the input box with the current value. Then, replace it with an array of strings containing all attributes in your index except the primary key:
+Find your primary key, then click on the bin icon:
 
-![The "Searchable attributes" configuration section showing the new setting value: \["title","author","genre","price"\].](/assets/images/cloud-index-settings/05-searchable-attributes-filled.png)
+![The same "Searchable attributes" list as before, with the bin-shaped "delete" icon highlighted.](/assets/images/cloud-index-settings/05-searchable-attributes-delete.png)
 
-Two buttons will appear on the bottom of the screen when you start editing a field. Once you're finished editing a setting, click on "Save":
+Meilisearch will display a pop-up window asking you to confirm you want to remove the attribute from the searchable attributes list. Click on "Yes, remove attribute":
 
-![The "Searchable attributes" configuration section showing two buttons at the bottom of the screen: "Cancel changes" and "Save".](/assets/images/cloud-index-settings/06-save.png)
+![A pop-up window over the index settings interface. It reads: "Are you sure you want to remove the attribute id?" Below it are two buttons: "Cancel" and "Yes, remove attribute".](/assets/images/cloud-index-settings/06-searchable-attributes-confirm-deletion.png)
 
-The button icon will change to a spinning wheel to indicate Meilisearch is processing the changes. Depending on the size of your dataset and the setting you have edited, this might take some time:
+Most updates to an index's settings will cause Meilisearch to re-index all its data. Wait a few moments until this operation is complete. You are not allowed to update any index settings during this time.
 
-![The "Searchable attributes" configuration section showing two buttons at the bottom of the screen. The "Save" button now has a spinning loading icon.](/assets/images/cloud-index-settings/07-saving.png)
+Once Meilisearch finishes indexing, the primary key will no longer appear in the searchable attributes list:
 
-Once Meilisearch is ready, the button should change to a green color. You will also see a notification pop-up on the right-hand side of your screen.
+![The same "Searchable attributes" list as before. It only contains five searchable attributes after removing the primary key.](/assets/images/cloud-index-settings/07-searchable-attributes-attribute-deleted.png)
 
-![The "Searchable attributes" configuration section showing two buttons at the bottom of the screen. The "Save" button has changed color and now reads "Saved".](/assets/images/cloud-index-settings/08-saved.png)
+If you deleted the wrong primary key, click on "Add attributes" to add it back to the list. You may also click on "Reset to default", which will bring back the searchable list to its original state when you first added your first document to this index:
```

Comment:
"If you delete the wrong primary key" make me feel there could be multiple primary keys by reading it. 

```suggestion
If you deleted the wrong attribute, click on "Add attributes" to add it back to the list. You may also click on "Reset to default", which will bring back the searchable list to its original state when you first added your first document to this index:
```